index.html

```diff
@@ -0,0 +1,309 @@
+       <div class="item">
+                <img src="./images/mob_Portfolio2.png" alt="img1">
+                <div class="info_projet">
+                    <h2>Tonic</h2>
+                    <div class="type">
+                        <p class="client">CANOPY</p>
+                        <img src="./images/circle.png" alt="">
+                        <p class="service">Back End Dev</p>
+                        <img src="./images/circle.png" alt="">
+                        <p class="service">2015</p>
+                    </div>
+                    <p class="description">
+                        A daily selection of privately personalized reads; 
+                        no accounts or sign-ups required.
+                    </p>
+                    <div class="languages">
+                        <p>html</p>
+                        <p>css</p>
+                        <p>javaScript</p>
+                    </div>
+                    <button>See Project</button>
+                </div>
+        </div>
+        <div class="item">
+                <img src="./images/mob_Portfoli1.png" alt="img1">
+                <div class="info_projet">
+                    <h2>Tonic</h2>
+                    <div class="type">
+                        <p class="client">CANOPY</p>
+                        <img src="./images/circle.png" alt="">
+                        <p class="service">Back End Dev</p>
+                        <img src="./images/circle.png" alt="">
+                        <p class="service">2015</p>
+                    </div>
+                    <p class="description">
+                        A daily selection of privately personalized reads;
+                        no accounts or sign-ups required.
+                    </p>
+                    <div class="languages">
+                        <p>html</p>
+                        <p>css</p>
+                        <p>javaScript</p>
+                    </div>
+                    <button>See Project</button>
+                </div>
+        </div>
+        <div class="item">
+                <img src="./images/mob_Portfolio3.png" alt="img1">
+                <div class="info_projet">
+                    <h2>Tonic</h2>
+                    <div class="type">
+                        <p class="client">CANOPY</p>
+                        <img src="./images/circle.png" alt="">
+                        <p class="service">Back End Dev</p>
+                        <img src="./images/circle.png" alt="">
+                        <p class="service">2015</p>
+                    </div>
+                    <p class="description">
+                        A daily selection of privately personalized reads;
+                        no accounts or sign-ups required.
+                    </p>
+                    <div class="languages">
+                        <p>html</p>
+                        <p>css</p>
+                        <p>javaScript</p>
+                    </div>
```

Comment:
Great job on styling this `section` 🎨  but the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") should be a list `ul` and each of them a `li`. This will help browsers understand your page a bit better. 😉

Reply:
[Mandatory] 

As mentioned by @didierganthier, better to use the `ul` and `li` tags in order to respect the semantics.

Please take [this link as a reference](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul).